lib/irb/extend-command.rb

```diff
@@ -197,6 +197,14 @@ def irb_context
         :irb_history, :History, "cmd/history",
         [:history, NO_OVERRIDE],
         [:hist, NO_OVERRIDE],
+      ],
+
+      [
+        :irb_exit_program, :ExitProgram, 'cmd/exit_program',
+        [:exit_program, NO_OVERRIDE],
+        [:quit_program, NO_OVERRIDE],
+        [:'exit-program', NO_OVERRIDE], # FIXME: This doesn't work properly; seems to match normal "exit" command
```

Comment:
This doesn't seem to work, but maybe we don't care?

I'd added these to match the commands in Pry, but I see we use `show_source` rather than `show-source`, so maybe we only want `exit_program` and `quit_program`.